src/runner.rs

```diff
@@ -2,13 +2,14 @@ use std::convert::From;
 use std::env;
 use std::error::Error;
 use std::fmt;
-use std::io::{Result as IoResult, Error as IoError};
+use std::io::{Result as IoResult, Error as IoError, ErrorKind};
 use std::path::{Path, PathBuf};
-use std::process::{Command, Stdio};
 use std::process;
-use mozprofile::profile::Profile;
+use std::process::{Command, Stdio};
+
 use mozprofile::prefdata::FIREFOX_PREFERENCES;
 use mozprofile::prefreader::PrefReaderError;
+use mozprofile::profile::Profile;
```

Comment:
I don't think that there's any sort order that would put `std` before `mozprofile` but `fmt` after `err`. AFAICT the normal style in Rust is just to sort all lines rather than trying to do the python thing of dividing up stdlib/other lib/project.